pro_tes/ga4gh/tes/service_info.py

```diff
@@ -0,0 +1,57 @@
+"""Controllers for the `/service-info route."""
+
+import logging
+from telnetlib import SE
+from typing import Dict
+
+from bson.objectid import ObjectId
+from foca.models.config import Config
+from flask import current_app
+from pymongo.collection import Collection
+from pro_tes.exceptions import (
+    NotFound,
+)
+
+logger = logging.getLogger(__name__)
+
+class ServiceInfo:
+    
+    def __init__(self) -> None:
+        """Class for TES API service info server-side controller methods.
+
+        Creates service info upon first request, if it does not exist.
+
+        Attributes:
+            config: App configuration.
+            foca_config: FOCA configuration.
+            db_client_service_info: Database collection storing service info
+                objects.
+            db_client_runs: Database collection storing workflow run objects.
```

Comment:
We are dealing with tasks here, so use `db_client_tasks`. Also update the description accordingly.